src/core/flate_stream.js

```diff
@@ -238,6 +238,11 @@ class FlateStream extends DecodeStream {
     return [codes, maxLen];
   }
 
+  endsStreamOnError(err) {
```

Comment:
How about making this private instead, since nothing should ever access it from the outside?
```suggestion
  #endStreamOnError(err) {
```

Reply:
I wonder if it's useful/helpful to warn here, wdyt ?

Reply:
You could change it to `info` instead, since that's not printed by default (but can still be enabled by changing the `verbosity`-option).